ci_framework/roles/use_install_yamls/tools/generate_install_yamls_playbooks.py

```diff
@@ -0,0 +1,31 @@
+# Python Script to generate playbooks
+import os
+import sys
+
+script_path = os.path.realpath(os.path.dirname(__file__))
+playbooks_dir = os.path.join(os.path.dirname(script_path), 'tasks')
+make_file = sys.argv[1]
+
+# Read the content of the makefile
+with open(make_file) as f:
+    content = f.read().split('\n')
+
+# Get all the make targets
+make_targets = []
+for data in content:
+    if data.startswith('.PHONY: '):
+        command = data.split('.PHONY: ')[1]
+        make_targets.append(command)
+
+# Generate playbooks
+for target in make_targets:
+    makefile = f'make_{target}.yaml'
+    with open(os.path.join(playbooks_dir, makefile), 'w') as f:
+        f.write(f'''
+---
+- name: Run `make {target}`
+  community.general.make:
```

Comment:
use the provided `ci_make` action plugin instead. It will generate reproducer make calls.

Reply:
```suggestion
  ci_make:
    output_dir: "{{ cifmw_use_install_yamls_basedir }}/artifacts }}"
```

And we may also set the "dry_run" flag if wanted, though it's probably not really useful at this stage.